src/public/NapiJsiRuntime.cpp

```diff
@@ -626,6 +627,11 @@ facebook::jsi::PropNameID NapiJsiRuntime::createPropNameIDFromString(const faceb
   return MakePointer<facebook::jsi::PropNameID>(uniqueStr);
 }
 
+facebook::jsi::PropNameID NapiJsiRuntime::createPropNameIDFromSymbol(const facebook::jsi::Symbol &sym) {
+  CHECK_ELSE_CRASH(false, "Not implemented");
```

Comment:
> CHECK_ELSE_CRASH(false, "Not implemented"); [](http://example.com/codeflow?start=1&length=44)

Please note that I am deleting this file in
[https://github.com/microsoft/v8-jsi/pull/122](https://github.com/microsoft/v8-jsi/pull/122)

The real fix for this code must go to RNW implementation: 
[https://github.com/microsoft/react-native-windows/blob/main/vnext/Microsoft.ReactNative.Cxx/JSI/NodeApiJsiRuntime.cpp](https://github.com/microsoft/react-native-windows/blob/main/vnext/Microsoft.ReactNative.Cxx/JSI/NodeApiJsiRuntime.cpp)

Reply:
Please ignore this comment: based on the feedback I had restored the deleted files.
Let me know if I can help with implementing the change in the NapiJsiRuntime.cpp.

Reply:
okay awesome! I actually have a branch that fixes the changes in react-native-windows (https://github.com/TatianaKapos/react-native-windows/commit/43293c11e96f3832ceb861d4905886e18ef6ba84) and will make a PR once these gets merged in.